### PR TITLE
add references from hoeft-et-al-icaps2024

### PR DESCRIPTION
**Pull request by phoeft670**

### CODE REVIEW

**Comment by phoeft670** [comment]
@jendrikseipp

File: literatur.bib [hunk 1]
```diff
@@ -1072,6 +1072,14 @@ @Misc{battaglia-et-al-arxiv2018
   howpublished = "arXiv:1806.01261 [cs.LG]"
 }
 
+@Book{bazaraa-et-al-2009,
+  title =         "Linear Programming and Network Flows",
+  author =        "Bazaraa, M.S. and Jarvis, J.J. and Sherali, H.D.",
```
Comment (jendrikseipp) [inline literatur.bib]: We use first name before last name.

Comment (jendrikseipp) [reply literatur.bib]: First names should be in long form if available and a space between them.

File: literatur.bib [hunk 2]
```diff
@@ -6020,6 +6037,15 @@ @InProceedings{furtak-buro-ijcai2011
   pages =        "534--539"
 }
 
+@Article{gal-ors1986,
+  author =       "Gal, T.",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  author =       "T. Gal",
```

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

File: literatur.bib (hunk 1 above)
Comment (phoeft670) [reply literatur.bib]: updated in [fe33d8b](https://github.com/aibasel/bib/pull/3/commits/fe33d8b6a6b27ae298e9e4c77343680fde87a04b)

**Pull request merged**